src/rust/src/declarative_asn1/types.rs

```diff
@@ -64,15 +65,40 @@ pub struct AnnotatedTypeObject<'a> {
     pub value: pyo3::Bound<'a, pyo3::PyAny>,
 }
 
-#[pyo3::pyclass(module = "cryptography.hazmat.bindings._rust.asn1")]
-#[derive(Clone, Debug)]
-pub struct Annotation {}
+#[pyo3::pyclass(frozen, module = "cryptography.hazmat.bindings._rust.asn1")]
+#[derive(Debug)]
+pub struct Annotation {
+    #[pyo3(get)]
+    pub(crate) default: Option<pyo3::Py<Default>>,
+}
 
 #[pyo3::pymethods]
 impl Annotation {
     #[new]
-    fn new() -> Self {
-        Self {}
+    #[pyo3(signature = (default = None))]
+    fn new(default: Option<pyo3::Py<Default>>) -> Self {
+        Self { default }
+    }
+
+    #[pyo3(signature = ())]
+    fn is_empty(&self) -> bool {
+        self.default.is_none()
+    }
+}
+
+#[pyo3::pyclass(frozen, module = "cryptography.hazmat.bindings._rust.asn1")]
```

Comment:
Sorry, should have asked this earlier: why do we hvae a `Default` type at all? Why not just have the default field on the Annotation be a PyAny?